tableView/ViewController.swift

```diff
@@ -1,19 +1,161 @@
-//
-//  ViewController.swift
-//  tableView
-//
-//  Created by Mark Wilhelm on 01.06.2022.
-//
-
 import UIKit
 
-class ViewController: UIViewController {
+struct Section {
+    let title: String
+    let options: [SettingsOptionType]
+}
+
+enum SettingsOptionType {
+    case staticCell(model: SettingsOption)
+    case switchCell(model: SettingsSwitchOption)
+}
+
+struct SettingsSwitchOption {
+    let title: String
+    let icon: UIImage?
+    let iconBackgroundColor:  UIColor
+    let handler: (() -> Void)
+    var isOn: Bool
+}
+
+struct SettingsOption {
+    let title: String
+    let icon: UIImage?
+    let iconBackgroundColor:  UIColor
+    let labelCondition: String
+    let handler: (() -> Void)
+}
+
+class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
```

Comment:
Если тебе нужно подписать класс на протокол, то делать это нужно в расширениях, чтобы не награмождать основной класс
```suggestion
class ViewController: UIViewController {
```